### DISCUSSION TITLE
Version 7.x with PowerShell ISE

### DISCUSSION THREAD

**Discussion by radascuta** [PowerShell/PowerShell#14955] [Q&A] [answered]
Is there a way to make version 7 work with PowerShell ISE?

**Comment by doctordns** [comment, marked as answer]
No. At least in any sort of supported way.

Use VS Code as a replacement.

**Comment by 9072997** [comment]
Check out the "PowerShell ISE Add-On Command" section [here](https://old.ironmansoftware.com/using-powershell-core-6-and-7-in-the-windows-powershell-ise/#:~:text=PowerShell%20ISE%20Add-On%20Command)

**Reply by doctordns** [reply]
Is this supported???
In my view,  once I got converted to VS code,  I haven't looked back. :-)

**Reply by 9072997** [reply]
Unless you have some sort of special support agreement with Microsoft, all of PowerShell Core is unsupported, right? To quote the license:
>THE SOFTWARE IS PROVIDED \*AS IS\*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.

Personally, I have been using VS Code even before I switched from Windows PowerShell to PowerShell Core. I prefer it. The community seems to be moving that way, so going forward things are likely to get better on VS Code, whereas the solution I linked to is the code equivalent of duct tape. Considering how simple it is though, it works surprisingly well.
